### DISCUSSION TITLE
More LHR unlock soon ?

### DISCUSSION THREAD

**Discussion by minerminat** [NebuTech/NBMiner#727] [Ideas]
Really appreciate all the work on unlocking LHR GPUs to 74% but would really love to get more unlocking on our LHR cards. Any plans on working on this anytime soon ?

**Comment by Badrick16** [comment]
I would really appreciate to less power consumption. If they increase the hashrate that means higher Network hashrate and increased difficulty, So overall we get less ETH in timewise even our some card has improved hasrhate.
